pontoon/base/static/js/main.js

```diff
@@ -479,17 +479,24 @@ $(function() {
 
   // Menu sort
   $('.menu .sort span').click(function (e) {
-    function val(index, el) {
+    function val(index, el, type) {
```

Comment:
You don't need to pass index here I think, it's hoisted to the click handler function so it's accessible within this function.